shared/elements/gaia-header/dist/gaia-header.js

```diff
@@ -1127,8 +1127,9 @@ module.exports = component.register('gaia-header', {
   sumButtonWidths: function(buttons) {
     var defaultWidth = 50;
     var sum = buttons.reduce((prev, button) => {
+      var isEmpty = !button.children.length && !button.textContent;
```

Comment:
Is trim() necessary? From MDN, whitespace is considered [1]. But I should make sure it has dataset.icon.

[1] https://developer.mozilla.org/zh-TW/docs/Web/CSS/:empty